man/systemd.exec.xml

```diff
@@ -1745,6 +1745,21 @@ BindReadOnlyPaths=/var/lib/systemd</programlisting>
         <xi:include href="system-or-user-ns.xml" xpointer="singular"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>MemoryKSM=</varname></term>
+
+        <listitem><para>Takes a boolean argument. When set, it enables KSM (kernel samepage merging) for
+        the processes. KSM is a memory-saving de-duplication feature. Anonymous memory pages with identical
+        content can be replaced by a single write-protected page. This feature should only be enabled for
+        jobs that share the same security domain. The feature is described in this
+        <ulink url="https://docs.kernel.org/admin-guide/mm/ksm.html">document</ulink>.
```

Comment:
please place the doc title in the link text. when the man pages are built a table of links is generated at the bottom, and it's useless if it just says "document" there.

maybe just say:

```
For details, see <ulink url="https://docs.kernel.org/admin-guide/mm/ksm.html">Kernel Samepage Merging</ulink> in the kernel documentation.
```
or so

Reply:
Done.